src/framework/audio/qml/MuseScore/Audio/KnobControl.qml

```diff
@@ -159,4 +159,52 @@ Dial {
     Component.onCompleted: {
         backgroundCanvas.requestPaint()
     }
+
+    onMoved: {
+        newValueRequested(value)
+    }
+
+    MouseArea {
+        id: mouseArea
+        anchors.fill: parent
+        onDoubleClicked: {
+            root.newValueRequested(0)
+        }
+
+        // The MouseArea steals mouse press events from the slider.
```

Comment:
I agree we can do this in a future PR. Maybe we should do this via C++, like the QML components also do. It would be great if we could subclass QML components, but we can't because they're in private headers...

Reply:
If you're speaking about QtQuick.Controls, then there is no need to deal with this library here. The logic around this "new" component is pretty simple.

However, if it'll be necessary, we can wrap our own implementation into Control qml type (which is a base (C++) component for all QtQuick.Controls)